pixeltable/env.py

```diff
@@ -576,6 +576,13 @@ def _create_engine(self, time_zone_name: str, echo: bool = False) -> None:
             assert isinstance(tz_name, str)
             self._logger.info(f'Database time zone is now: {tz_name}')
             self._default_time_zone = ZoneInfo(tz_name)
+            if self.is_using_cockroachdb:
+                # This should have been set when the database was created,
+                # but if the db predates the fix or we are running as root, set it now
```

Comment:
Better to do this as a migration? Or just manually issue the ALTER command in our existing DBs? (if that's possible?) Then this could become just an assertion.